From 5189e9b511248e0b1e429893ab8c1138ca3b0b48 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Wed, 8 Jul 2020 17:03:11 +0800
Subject: [PATCH] drivers: devfreq: rockchip_dmc: export dmc symbol

EXPORT_SYMBOL rockchip_dmcfreq_vop_bandwidth_update and
rockchip_dmcfreq_vop_bandwidth_request

Change-Id: Ifb41d9e7885627a3c085fb403289bd3469d073c9
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/devfreq/rockchip_dmc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/devfreq/rockchip_dmc.c b/drivers/devfreq/rockchip_dmc.c
index fcf118a2c620..f2a7ac939bae 100644
--- a/drivers/devfreq/rockchip_dmc.c
+++ b/drivers/devfreq/rockchip_dmc.c
@@ -3276,6 +3276,7 @@ void rockchip_dmcfreq_vop_bandwidth_update(struct devfreq *devfreq,
 	if (target > vop_last_rate)
 		rockchip_dmcfreq_update_target(dmcfreq);
 }
+EXPORT_SYMBOL(rockchip_dmcfreq_vop_bandwidth_update);
 
 int rockchip_dmcfreq_vop_bandwidth_request(struct devfreq *devfreq,
 					   unsigned int bw_mbyte)
@@ -3305,6 +3306,7 @@ int rockchip_dmcfreq_vop_bandwidth_request(struct devfreq *devfreq,
 	else
 		return -EINVAL;
 }
+EXPORT_SYMBOL(rockchip_dmcfreq_vop_bandwidth_request);
 
 static int devfreq_dmc_ondemand_func(struct devfreq *df,
 				     unsigned long *freq)
-- 
2.35.3

